samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/app_gpio.h

```diff
@@ -0,0 +1,17 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _APP_GPIO_H
+#define _APP_GPIO_H
+
+/* GPIO */
+extern struct device *button_device[4];
+extern struct device *led_device[4];
+
+void gpio_init(void);
```

Comment:
gpio_* is a reserved prefix for the kernel's GPIO subsystem. Since this is a public symbol it'd be safer to call it something else, to avoid confusion and potential future conflicts. Call it e.g. app_gpio_init(). That would also make it consistent with the header file it's located in.

Reply:
done !!